akka-projection-grpc/src/main/scala/akka/projection/grpc/consumer/GrpcQuerySettings.scala

```diff
@@ -30,7 +30,7 @@ object GrpcQuerySettings {
       "Configuration property [stream-id] must be an id exposed by the producing side but was undefined on the consuming side.")
 
     val additionalHeaders: Option[Metadata] = {
-      import scala.jdk.CollectionConverters._
+      import akka.util.ccompat.JavaConverters._
```

Comment:
To be honest, I don't understand how this could have compiled with 2.12. I tried locally, and it did compile with scala.jdk. As far as I can see it's not in 2.12 https://www.scala-lang.org/api/2.12.17/scala/index.html

However, we use ccompat in all other places so changed it.